app/routes/file-router.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const categoryController = require('../controllers/file-controller');
```

Comment:
Why the name `categoryController`. Please give a name related to the controller's behavior.